core/vm/analysis.go

```diff
@@ -76,6 +76,14 @@ func codeBitmapInternal(code, bits bitvec) bitvec {
 	for pc := uint64(0); pc < uint64(len(code)); {
 		op := OpCode(code[pc])
 		pc++
+
+		// handle super instruction.
+		step, processed := codeBitmapForSI(code, pc, op, &bits)
+		if processed {
+			pc += step
+			continue
+		}
+
```

Comment:
it is ok because when optimization is disabled, the codebitmap here does't encounter any super instruction in this method, so the method takes no effect. On the otherside, it is hard in this go file to get the VMConfig and hence hard to add logic to know whether the optimization is disabled. So I suggest we leave it there.